<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>711</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>your selections</string>
    </property>
    <widget class="QLabel" name="batman_value">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="bowler_value">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="allrounde_value">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="wicketkeeper_value">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>40</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>110</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bowlers (BLW)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>124</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>AllRounders (AR)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>40</y>
       <width>100</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Batman (BAT)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>505</x>
       <y>41</y>
       <width>145</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Wicket-Keeper (WK)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>301</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>points available</string>
    </property>
    <widget class="QListWidget" name="selections_list_widget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>256</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="points_available_value">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>242</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="BAT_radio_btn">
        <property name="text">
         <string>BAT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="BOW_radio_btn">
        <property name="text">
         <string>BLW</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="AR_radio_btn">
        <property name="text">
         <string>AR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="WK_radio_btn">
        <property name="text">
         <string>WK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>301</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>points used</string>
    </property>
    <widget class="QListWidget" name="team_added_list_widget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>256</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="points_used_value">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="TeamName">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Team Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="Team_name_value">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="arrow_symbol">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>360</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>manage teams</string>
    </property>
    <addaction name="actionnew_team"/>
    <addaction name="actionopen_team"/>
    <addaction name="actionsave_team"/>
    <addaction name="actionevaluate_team"/>
   </widget>
   <addaction name="menumenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionnew_team">
   <property name="text">
    <string>new team</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionopen_team">
   <property name="text">
    <string>open team</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionsave_team">
   <property name="text">
    <string>save team</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionevaluate_team">
   <property name="text">
    <string>evaluate team</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
